### PR TITLE
Squash docker layers to make the scope image fetch nicer.

### LINKED ISSUE #82: Scope should use fewer docker layers
```
vagrant@ubuntu-14:~/src/github.com/weaveworks/scope$ ./scope launch
Unable to find image 'weaveworks/scope:latest' locally
Pulling repository weaveworks/scope
8987330a95fa: Pulling image (latest) from weaveworks/scope
8987330a95fa: Pulling image (latest) from weaveworks/scope, endpoint: https://registry-1.docker.io/v1/
8987330a95fa: Pulling dependent layers
511136ea3c5a: Download complete
a8be93256def: Download complete
2db66c93c72e: Pulling metadata
2db66c93c72e: Pulling fs layer
2db66c93c72e: Download complete
98b0c7b63452: Pulling metadata
98b0c7b63452: Pulling fs layer
98b0c7b63452: Download complete
22618500139b: Pulling metadata
22618500139b: Pulling fs layer
22618500139b: Download complete
bd7d0b16def3: Pulling metadata
bd7d0b16def3: Pulling fs layer
bd7d0b16def3: Download complete
6ff49fd79084: Pulling metadata
6ff49fd79084: Pulling fs layer
6ff49fd79084: Download complete
4371bc9dee57: Pulling metadata
4371bc9dee57: Pulling fs layer
4371bc9dee57: Download complete
9819431aea96: Pulling metadata
9819431aea96: Pulling fs layer
9819431aea96: Download complete
02289b89fa78: Pulling metadata
02289b89fa78: Pulling fs layer
02289b89fa78: Download complete
281e7cdd2aae: Pulling metadata
281e7cdd2aae: Pulling fs layer
281e7cdd2aae: Download complete
8987330a95fa: Pulling metadata
8987330a95fa: Pulling fs layer
8987330a95fa: Download complete
8987330a95fa: Download complete
Status: Downloaded newer image for weaveworks/scope:latest
bc267a103940127e67b2dda613f77a80400c104c5ecb08c2c5bf185215ff50bf
```

### PR DESCRIPTION
**Pull request by tomwilkie**
Fixes #82

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
-ADD ./run-probe /etc/service/probe/run
-
-RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories && apk add --update runit
-
+RUN echo "http://dl-4.alpinelinux.org/alpine/edge/testing" >>/etc/apk/repositories && \
+	apk add --update runit && \
+	rm -rf /var/cache/apk/*
+COPY ./app ./probe ./entrypoint.sh /home/weave/
+COPY ./run-app /etc/service/app/run
+COPY ./run-probe /etc/service/probe/run
```
Comment (peterbourgon) [inline docker/Dockerfile]: Why the change from ADD to COPY? Does ADD generate new layers?

File: Makefile [hunk 2]
```diff
@@ -1,8 +1,8 @@
-.PHONY: all static clean
+.PHONY: all deps static clean
 
 # If you can use Docker without being root, you can `make SUDO= <target>`
 SUDO=sudo
-
+DOCKER_SQUASH=$(shell which docker-squash)
```
Comment (peterbourgon) [inline Makefile]: What is docker-squash? It's not part of any of my Docker installs...

**Comment by peterbourgon** [comment]
Those points of confusion aside, :dango:

File: docker/Dockerfile (hunk 1 above)
Comment (tomwilkie) [inline docker/Dockerfile]: COPY is preferred https://docs.docker.com/articles/dockerfile_best-practices/#add-or-copy

File: Makefile (hunk 2 above)
Comment (tomwilkie) [inline Makefile]: Its a new tool which squashes down docker layers.  I've added it to make deps to install it for you.
